Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -333,14 +333,17 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             )
         }
 
-        var closeAfterRead = DelayableAction(target: cache) { cache in
```

Comment:
we may be able to deprecate DelayableAction with this, I think this was the only call site?

as a side note, all this code should be refactored to async/await to reduce complexity

Reply:
It's used in `evaluateManifest()` as well.